css/main.css

```diff
@@ -0,0 +1,248 @@
+/* ------------------Home Page css---------------------- */
+/* PROBLEM: MOBILE VIEW KNOCKS CENTER CONTAINER OUT AND ENDS UP PUSHING NEXT BUTTON OFF SCREEN ON ALL PAGES (need to look at code, and need help with media queries), also ANIMATION MAKES CONTENT POP OUT OF THE BORDER AREA, BUTTON HOVER NOT WORKING*/
+
+/*the body is given a margin of zero to eliminate any margins, and a black background color so entire page appears black*/
+body {
+  display: flex; /*creates a flex container in the body element*/
+  justify-content: center; /*centers content in a flex container*/
+  border: 10px rgb(166, 137, 180) solid;
+  border-radius: 6px; /*rounds the edges of the border so they are not sharp*/
+  max-width: 80vw; /*sets a max view width on viewer's screen similarly to percent, responsive*/
+  padding: 10px; /*padding around body element*/
+  background-color: black;
+}
+
+/* a container holding the page elements*/
+#containerblue {
+  display: flex;
+  flex-direction: column; /*sets the flex direction to column - it would be row by default. widths and heights will apply to the column as the main axis is horizontal for column flex direction*/
+  justify-content: stretch;
+  align-items: center;
+  flex: 1; /*allows container tag to take up entire container width*/
+  margin: 0; /*no margin around the container*/
+  max-width: 80vw; /*sets height and width to specified view width*/
+  height: 40vw;
+  background-color: navy;
+  padding: 5px;
+}
+
+/*Gives the h1 content font properties and some padding*/
+h1 {
+  font-size: 30px;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  color: rgb(212, 42, 228);
+  padding-bottom: 20px;
+}
+
+/*css to set  wrap image in a background color of aqua and box shadow and set animation timing*/
+#homeimg {
+  text-align: center;
+  width: 30vw;
+  margin-bottom: 15px;
+  background-color: aqua;
+  box-shadow: 0.5vw 1vh; /*controlling view width and height of box shadow*/
+  padding: 3vh 5vw 2vh 5vw; /*top, left, bottom, right padding*/
+  transition: all 0.5s ease; /*what is being targeted, time, animation timing-accelerates*/
+  transition-property: width, background-color; /*properties that will transition with the action (pseudoclass/id)*/
+}
+/*--home page image hover animation that causes image to grow and background color to change to pink when user hovers over it then it returns to normal when they are no longer hovering--*/
+#homeimg:hover {
+  width: 40vw; /*original width was 35, this makes it pop out 5vw with hover*/
+  background-color: rgb(
+    243,
+    49,
+    178
+  ); /*color that background will change to with hover*/
+  transition-property: width, background-color; /*properties that will transition with the action (pseudoclass/id)*/
+  transition: all 0.5s ease; /*what is being targeted, time, animation timing-accelerates*/
+}
```

Comment:
This is really cool! I don't think Bob is counting hover effects as animations (only things with key-frames) but I'll ask him about this since it's more than just a normal hover 👍 Again... VERY COOL!